lib/trino-parquet/src/test/java/io/trino/parquet/writer/TestParquetSchemaConverter.java

```diff
@@ -123,6 +125,50 @@ public void testMapKeyRepetitionLevel()
         assertThat(valueType.isRepetition(OPTIONAL)).isTrue();
     }
 
+    /**
+     * This test ensures the parquet writer matches the parquet spec for lists.
+     *
+     * From https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists
+     *
+     * Lists
+     * LIST is used to annotate types that should be interpreted as lists.
+     *
```

Comment:
I don't think this blank line is necessary

Reply:
ping